scripts/README.md

```diff
@@ -0,0 +1,46 @@
+# Tools
+
+## Update README (`update-readme.py`)
+
+This script automatically updates the `README.md` file in the repository root.
+The content is build by itterating over files matching `revpi-hat-PR*R*.json` and gathering their product information.
```

Comment:
itterating -> iterating

`The content is build by iterating over files matching `revpi-hat-PR*R*.json` and gathering their product information.`